junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/Constants.java

```diff
@@ -187,6 +205,205 @@
 	public static final String PARALLEL_CONFIG_CUSTOM_CLASS_PROPERTY_NAME = PARALLEL_CONFIG_PREFIX
 			+ CONFIG_CUSTOM_CLASS_PROPERTY_NAME;
 
+	/**
+	 * Property name used to set the default timeout for all testable and
+	 * lifecycle methods.
+	 *
+	 * <p>The value of this property will be used unless overridden by a more
+	 * specific property or a {@link org.junit.jupiter.api.Timeout @Timeout}
+	 * annotation present on the method or an enclosing test class (for testable
+	 * methods).
+	 *
+	 * <p>Please refer to the <a href="#supported-values-timeouts">class
```

Comment:
Does that work for linking to the `<h3>` element?

I thought one had to create an _anchor_ explicitly.

But... if that works like that, then cool.

Reply:
I tried it and it worked.